packages/manager-react-components/src/components/Modal/Modal.component.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { Ref } from 'react';
+import {
+  OdsButton,
+  OdsModal,
+  OdsText,
+  OdsSpinner,
+} from '@ovhcloud/ods-components/react';
+import {
+  ODS_BUTTON_VARIANT,
+  ODS_MODAL_COLOR,
+  ODS_BUTTON_COLOR,
+  ODS_TEXT_PRESET,
+  ODS_SPINNER_SIZE,
+} from '@ovhcloud/ods-components';
+
+export enum ModalTypeEnum {
```

Comment:
```suggestion
export enum MODAL_TYPE {
```

Reply:
Enum name `MODAL_TYPE` must match one of the following formats: PascalCaseeslint[@typescript-eslint/naming-convention](https://github.com/typescript-eslint/typescript-eslint/blob/v4.33.0/packages/eslint-plugin/docs/rules/naming-convention.md)

Reply:
But we can use directly ODS_MODAL_COLOR like suggest your second comment for avoid mapping

Reply:
it's fixed